Barotrauma/BarotraumaShared/SharedSource/Items/Components/Holdable/RangedWeapon.cs

```diff
@@ -138,6 +183,9 @@ public Vector2 TransformedBarrelPos
         private float currentChargeTime;
         private bool tryingToCharge;
 
+        private Character prevUser;
+        private int burstPos;
```

Comment:
Not clear what this means, needs a more descriptive name and/or comment. I think something like `burstCounter` or `burstIndex` would be more clear than `burstPos`.